_posts/0000-01-02-blackeye735.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+"Welocome to the posts section of github-slideshow repository.It is a learning process for advancement in Github labs."
```

Comment:
In this repository we are going to have an idea of the github branching and pull or merge requests.